administrator/components/com_menus/views/items/tmpl/modal.php

```diff
@@ -0,0 +1,134 @@
+						</th>
+						<th width="1%" class="nowrap hidden-phone">
+							<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_ID', 'a.id', $listDirn, $listOrder); ?>
+						</th>
+					</tr>
+				</thead>
+				<tfoot>
+					<tr>
+						<td colspan="5">
+							<?php echo $this->pagination->getListFooter(); ?>
+						</td>
+					</tr>
+				</tfoot>
+				<tbody>
+				<?php // @todo Neeeds the same columns as normla view ?>
+				<?php
+				$iconStates = array(
+					-2 => 'icon-trash',
+					0 => 'icon-unpublish',
+					1 => 'icon-publish',
+					2 => 'icon-archive',
+				);
+				?>
+				<?php foreach ($this->items as $i => $item) : ?>
+					<?php if ($item->language && JLanguageMultilang::isEnabled())
+					{
+						$tag = strlen($item->language);
+						if ($tag == 5)
+						{
+							$lang = substr($item->language, 0, 2);
+						}
+						elseif ($tag == 6)
+						{
+							$lang = substr($item->language, 0, 3);
+						}
+						else {
+							$lang = "";
+						}
+					}
+					elseif (!JLanguageMultilang::isEnabled())
+					{
+						$lang = "";
+					}
+					?>
+					<tr class="row<?php echo $i % 2; ?>">
+						<td class="center">
+							<span class="<?php echo $iconStates[$this->escape($item->published)]; ?>"></span>
+						</td>
+						<td>
+							<a href="javascript:void(0)" onclick="if (window.parent) window.parent.<?php echo $this->escape($function);?>('<?php echo $item->id; ?>', '<?php echo $this->escape(addslashes($item->name)); ?>', '<?php echo $this->escape($item->catid); ?>', null, '<?php echo $this->escape(NewsfeedsHelperRoute::getNewsfeedRoute($item->id, $item->catid, $item->language)); ?>', '<?php echo $this->escape($lang); ?>', null);">
+							<?php echo $this->escape($item->name); ?></a>
+							<div class="small">
+								<?php echo JText::_('JCATEGORY') . ": " . $this->escape($item->category_title); ?>
+							</div>
+						</td>
+						<td class="small hidden-phone">
+							<?php echo $this->escape($item->access_level); ?>
+						</td>
+						<td class="small hidden-phone">
+							<?php if ($item->language == '*'):?>
+								<?php echo JText::alt('JALL', 'language'); ?>
+							<?php else:?>
+								<?php echo $item->language_title ? JHtml::_('image', 'mod_languages/' . $item->language_image . '.gif', $item->language_title, array('title' => $item->language_title), true) . '&nbsp;' . $this->escape($item->language_title) : JText::_('JUNDEFINED'); ?>
+							<?php endif;?>
+						</td>
+						<td class="hidden-phone">
+							<?php echo (int) $item->id; ?>
+						</td>
+					</tr>
+				<?php endforeach; ?>
+				</tbody>
+			</table>
+		<?php endif; ?>
+
+		<input type="hidden" name="task" value="" />
+		<input type="hidden" name="boxchecked" value="0" />
+		<input type="hidden" name="forcedLanguage" value="<?php echo $app->input->get('forcedLanguage', '', 'CMD'); ?>" />
+		<?php echo JHtml::_('form.token'); ?>
+
+	</form>
+</div>
```

Comment:
yes, when it's ready we will make a PR in the core :D

we need this in this project so we have to do it ... for the core